tests/test_example_dags.py

```diff
@@ -24,6 +24,7 @@
 
 MIN_VER_DAG_FILE: dict[str, list[str]] = {
     "2.4": ["cosmos_seed_dag.py"],
+    "3.0": ["performance_dag.py"],
```

Comment:
Just so it's clear that the peformance dag doesn't require Airflow v3.0, what do you think about instead having another constant like `IGNORED_DAG_FILES = ["performance_dag.py"]` and writing those to the ignore file?